pkg/apis/features/features.go

```diff
@@ -0,0 +1,21 @@
+package features
+
+import (
+	"context"
+
+	tektonconfig "github.com/tektoncd/pipeline/pkg/apis/config"
+)
+
+// SetFeatureFlag sets the feature flag for Tekton parsers in the context.
+func SetFeatureFlag(ctx context.Context) context.Context {
```

Comment:
This is gonna sound weird, and it might be "too complex", but could we "get the configmap" from upstream and set those in context accordingly to the pipeline instance configuration instead ?

Reply:
we are not necessary running on cluster with tkn pac resolve... but does it matter to get the right configuration?, since its only for validation, and would further fail when applying on cluster if they don't have that setting

Reply:
Good point on `tkn pac resolve`. I think we most likely don't need to get the absolute right configuration for 99% of the time, but we may end-up with an error of "invalid" at some point because of a feature-flag being on in the cluster and "off" in the code. But I think we will see if that's ever the case 😅.

Reply:
but that just an optimization since this anyway would fail when applying,  it's a user error to write whatever they want if that's not supported on their k8 platform it's really their problem, we can only be helpful and let them know...